pkg/render/render.go

```diff
@@ -128,13 +128,18 @@ func loadTemplates(tmpl *template.Template, root string) error {
 	})
 }
 
+func unescape(s string) template.HTML {
```

Comment:
Nit: I'd prefer to name this `safeHTML` or something instead? It's possible we might want to do the same for JS or CSS in the future.

Reply:
done. added a doc comment